src/sentry/api/base.py

```diff
@@ -88,13 +87,6 @@
 audit_logger = logging.getLogger("sentry.audit.api")
 api_access_logger = logging.getLogger("sentry.access.api")
 
-DEFAULT_SLUG_PATTERN = r"^[a-z0-9_\-]+$"
-NON_NUMERIC_SLUG_PATTERN = r"^(?![0-9]+$)[a-z0-9_\-]+$"
-DEFAULT_SLUG_ERROR_MESSAGE = _(
-    "Enter a valid slug consisting of lowercase letters, numbers, underscores or hyphens. "
-    "It cannot be entirely numeric."
-)
```

Comment:
Removed `DEFAULT_SLUG_PATTERN` b/c we don't need it anymore. Moved the other 2 constants [here](https://github.com/getsentry/sentry/pull/59487/files#diff-8b06a256e96e39f9b30daf60bf9444f63a5ba02274e9e64853012ee3f5250b39R9-R13)